romancal/refpix/data.py

```diff
@@ -375,8 +385,19 @@ def remove_trends(self) -> ChannelView:
                 if x_vals.size < 1 or y_vals.size < 1:
                     continue
 
-                # Perform the fit using a 1st order polynomial, (i.e. linear fit)
-                m, b = np.polyfit(x_vals, y_vals, 1)
+                # # Perform the fit using a 1st order polynomial, (i.e. linear fit)
+                # m, b = np.polyfit(x_vals, y_vals, 1)
+
```

Comment:
Remove the commented out code and add a small explanation of what is going on here.